src/infi/clickhouse_orm/database.py

```diff
@@ -410,6 +410,8 @@ def _get_server_version(self, as_tuple=True):
         except ServerError as e:
             logger.exception('Cannot determine server version (%s), assuming 1.1.0', e)
             ver = '1.1.0'
+        # :TRICKY: Altinity cloud uses a non-numeric suffix for the version, which this removes.
```

Comment:
LGTM

Non-blocking: I'm new to this repo but can we add a test case for this? Also, should we use something like https://packaging.pypa.io/en/latest/version.html instead?

Reply:
I'd love to add a test, but sadly adding that support seems like would require a lot of work right now. Going to punt on this.